File: comms/bulk-messaging/sender-pools/remove-sender-from-pool/remove-sender-from-pool.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senderPools->removeSender(
    'comms_senderpool_01h9krwprkeee8fzqspvwy6nq8',
    'comms_sender_01h9krwprkeee8fzqspvwy6nq8',
);

File: comms/bulk-messaging/sender-pools/remove-sender-from-pool/remove-sender-from-pool.cs
using Twilio.Comms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.SenderPools.RemoveSenderAsync(
            "comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
            "comms_sender_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}
